swlor2_mdl/dr_lehon01.mdl

```diff
@@ -0,0 +1,4672 @@
+# NeverBlender 4.0.0|Blender 4.0.2|2023-12-25
```

Comment:
More compilation stuff. Make sure you're looking at all the individual files here so you know what's been compiled and what hasn't: https://github.com/zunath/SWLOR_Haks/pull/241/files